### ISSUE TITLE
TikTokLogoFill Icon has extra padding on the right

### ISSUE THREAD

**Issue by pr-1** [phosphor-icons/flutter#10]
I am using phosphor icons in one of my projects and the tiktok filled icon has extra padding on the right.

Code -> 

```dart
CircleAvatar(
                    backgroundColor: Colors.black,
                    child: Icon(PhosphorIcons.tiktokLogoFill, color: Colors.white,),
                  )
```
![WhatsApp Image 2022-02-28 at 11 00 45 AM](https://user-images.githubusercontent.com/26018750/155936577-2c787f0e-2593-4d83-a748-749f4dd05f7d.jpeg)

When I use same thing with any other icon it gives exact results

```dart
CircleAvatar(
                    backgroundColor: Colors.black,
                    child: Icon(PhosphorIcons.tiktokLogo, color: Colors.white,),
                  )
```

![WhatsApp Image 2022-02-28 at 11 00 57 AM](https://user-images.githubusercontent.com/26018750/155936709-f2d416ef-7b42-4d3d-b76e-72f166cf0910.jpeg)
 Can you please check this out and let me know how to fix this ? Would be happy to open a PR

**Comment by pr-1** [comment]
@rektdeckard Thanks for the great Icon set 🔥. Can you have a look at this ?

**Comment by rektdeckard** [comment]
Appears to be same root cause as https://github.com/phosphor-icons/phosphor-home/issues/103, using regions of Unicode reserved for special characters. Fix there will fix in flutter package as well.

**Comment by rektdeckard** [comment]
Fixed in 2.0.0.
